conv/conv.tex

```diff
@@ -0,0 +1,116 @@
+\documentclass[utf8x,hyperref={pdfpagelabels=false}]{beamer}
+
+\usepackage[utf8x]{inputenc}
+\usepackage[OT1]{fontenc}
+\usepackage{graphicx}
+\usepackage{xcolor}
+\usepackage{amsmath}
+
+\usetheme{Malmoe}  % Now it's a beamer presentation with the lisa theme!        
+\usecolortheme{beaver}
+\setbeamertemplate{footline}[page number]
+\setbeamertemplate{navigation symbols}{}
+
+\title{Convolutions for neural networks}
+
+\author{%
+Arnaud Bergeron
+}
+\date{August 10, 2015}
+
+\begin{document}
+
+\begin{frame}[plain]
+ \titlepage
+\end{frame}
+
+\setcounter{page}{1}
+
+\section{Convolution}
+
+% Definitions
+\begin{frame}
+Some vocabulary:
+\begin{description}[feature maps]
+\item[feature map] analogue to layer (the activations, not the weights) for a neural network.
+\item[channel] a single 2D "image" which composes a feature map.
+\item[filter] weight (as in layer weights).
```

Comment:
One filter is one 2D "image", that will be convolved with one input feature map, to give one component of the output feture map. There are (input_channels * output_channels) filters.